src/main/java/rx/observables/SwingObservable.java

```diff
@@ -163,6 +167,40 @@ public Boolean call(KeyEvent event) {
     }
 
     /**
+     * Creates an observable corresponding to item events.
+     * 
+     * @param component
+     *            The ItemSelectable to register the observable for.
+     * @return Observable emitting the item events for the given itemSelectable.
+     */
+    public static Observable<ItemEvent> fromItemEvents(ItemSelectable itemSelectable) {
+        return ItemEventSource.fromItemEventsOf( itemSelectable );
+    }
+    
+    /**
+     * Creates an observable corresponding to item selection events.
+     * 
+     * @param component
+     *            The ItemSelectable to register the observable for.
+     * @return Observable emitting the an item event whenever the given itemSelectable is selected.
+     */
+    public static Observable<ItemEvent> fromSelectionEvents(ItemSelectable itemSelectable) {
+        return ItemEventSource.fromItemEventsOf( itemSelectable ).filter( SELECTED );
+    }
+    
+    /**
+     * Creates an observable corresponding to item deselection events.
+     * 
+     * @param component
+     *            The ItemSelectable to register the observable for.
+     * @return Observable emitting the an item event whenever the given itemSelectable is deselected.
+     */
+    public static Observable<ItemEvent> fromDeselectionEvents(ItemSelectable itemSelectable) {
+        return ItemEventSource.fromItemEventsOf( itemSelectable ).filter( SELECTED );
```

Comment:
You're right, it is convenient. It's probably ok as long as it's not a static import.

Here are my suggestions before we merge:
* Fix the selected/deselected bugs (if you haven't already)
* Rename `fromSelectionEvents` and `fromDeselectionEvents` in SwingObservable to `fromItemSelectionEvents` and `fromItemDeselectionEvents`, respectively. (I think this will be a little more clear to users.)
* Get rid of the static import of `Predicate.SELECTED` in SwingObservable. You can still use the predicate if you want, or you could just filter via an an anonymous inner class (this is my preference).